app/BUILD.bazel

```diff
@@ -573,6 +573,9 @@ kt_android_library(
         ":dagger",
         ":listeners",
         ":resources",
+        "//app/src/main/java/org/oppia/android/app/viewmodel:observable_array_list",
+        "//app/src/main/java/org/oppia/android/app/viewmodel:observable_view_model",
+        "//app/src/main/java/org/oppia/android/app/viewmodel:view_model_provider",
```

Comment:
Can you check if 

`"//app/src/main/java/org/oppia/android/app/viewmodel:view_model_provider",`

is required in the deps here? Since that target was not part of the two srcs dependencies we removed.

Reply:
Yes, it is needed. I didn't include it at first and it gave some errors related to `ViewModelProvider` and `ViewModelBridgeFactory`

Reply:
Thanks! I'm assuming that there is some glob that stopped working once we turned viewmodel/ to a package. I think we can keep the dependency here for now.